Formula/homebank.rb

```diff
@@ -29,6 +29,12 @@ class Homebank < Formula
   depends_on "libsoup"
 
   def install
+    on_linux do
+      # Needed to find intltool (xml::parser)
+      ENV.prepend_path "PERL5LIB", Formula["intltool"].libexec/"lib/perl5"
+      ENV["INTLTOOL_PERL"] = Formula["perl"].bin/"perl"
```

Comment:
Given the number of formulae we're doing this in, it might be better to just do this in superenv whenever `intltool` is a dependency on Linux, no?

Reply:
Actually it does not hit that many formulae. It's just 4-5 that have a weird configure script that sets an extra INTLTOOL_PERL variable. All the other formulae that depend on intltool just work out of the box. So I think it's not worth it.

Here is the preliminary list:
```
gdl
gnome-themes-standard
goffice
gpredict
homebank
libgweather
```

Reply:
I think affecting five formulae is a fine bar for doing stuff in superenv, but it's up to you. How many other formulae use `intltool` but don't need this?

Reply:
54 - 6 -> 48. I know for sure that a lot do not need this workaround.
Not all have a linux bottle, so we did not test them all. I think we can wait and see, and come back to a superenv solution if we reach the 10 formula bar.